runtime/backtrace_nat.c

```diff
@@ -238,11 +238,19 @@ struct name_info {
   char name[1];
 };
 
+/* Extended version of name_info including location fields which didn't fit
+   in the main debuginfo word. */
+struct name_and_loc_info {
+  int32_t filename_offs;
+  uint32_t char_nums; /* Upper 16-bits for start; lower 16-bits for end */
```

Comment:
What's wrong with uint16_t?

Reply:
You mean I should let the _C compiler_ unpack my struct instead? 🤦‍♂️